src/test/java/com/wikia/webdriver/testcases/adstests/TestAdsRefreshing.java

```diff
@@ -16,8 +16,8 @@
   private static final String PATH = "SyntheticTests/Slots/RefreshOnView";
 
   private void scrollToSeeFMR(String slotName, AdsBaseObject page) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused private "scrollToSeeFMR" method. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AUnusedPrivateMethod)